notebooks/03_embedding_graph_structured_data_h2.ipynb

```diff
@@ -0,0 +1,888 @@
+{
```

Comment:
<p>Typo: yeilds -&gt; yields.</p><p> </p><p>The introduction is great!! Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done